src/Data/OpenApi.hs

```diff
@@ -119,6 +119,7 @@ module Data.OpenApi (
   -- ** Miscellaneous
   MimeList(..),
   URL(..),
+  SpecificationExtensions (..),
```

Comment:
Please copy this fix here :)

https://github.com/biocad/openapi3/pull/21/files#diff-72b8e7bcb5b302e7fe8867caf0eb76e8fc8c270acdcd5a6dc2fc1402a3904abcL187